apps/desktop-wallet/src/api/queries/tokenQueries.ts

```diff
@@ -156,14 +160,15 @@ export const nftDataQuery = ({ id, tokenUri, networkId, skip }: NFTQueryProps) =
 
 export const tokenQuery = ({ id, networkId, skip }: TokenQueryProps) =>
   queryOptions({
+    // eslint-disable-next-line @tanstack/query/exhaustive-deps
```

Comment:
For some reason the linter thinks that `fTList[id]` should be added as a dep, but that doesn't make sense.

Reply:
I noticed that in my next PR where I move this code to the shared package, the linter doesn't complain anymore..!